README.md

```diff
@@ -288,29 +288,34 @@ Basic proof of 'ipfs working' locally:
 
 SUBCOMMANDS
   BASIC COMMANDS
-    init          Initialize ipfs local configuration
-    add <path>    Add a file to ipfs
-    cat <ref>     Show ipfs object data
-    get <ref>     Download ipfs objects
+    init          Initialize local IPFS configuration
+    add <path>    Add a file to IPFS
+    cat <ref>     Show IPFS object data
+    get <ref>     Download IPFS objects
     ls <ref>      List links from an object
     refs <ref>    List hashes of links from an object
 
   DATA STRUCTURE COMMANDS
+    dag           Interact with IPLD DAG nodes
+    files         Interact with files as if they were a unix filesystem
+    object        Interact with dag-pb objects (deprecated, use 'dag' or 'files')
     block         Interact with raw blocks in the datastore
-    object        Interact with raw dag nodes
-    files         Interact with objects as if they were a unix filesystem
 
   ADVANCED COMMANDS
     daemon        Start a long-running daemon process
```

Comment:
"Start IPFS as a daemon (process)"

How long doesn't matter for this command to work

Reply:
https://github.com/ipfs/go-ipfs/pull/8010#discussion_r604383280